ui/src/translations/fr.json

```diff
@@ -179,10 +179,7 @@
   "start": "Début",
   "no_data_available_for_metrics": "Aucune donnée disponible pour afficher les Métriques",
   "global_health_explanation": "L'indicateur de santé generale frounit une information sur le status de votre Platform pour une periode donnée.\nLe statut des Volumes et des Noeuds, celui du Reseau et des Services sont supervisés.\nUn cercle de couleur verte signifie que la Platform est en bonne santé.\nUn cercle de couleur orange signifie que la Platform est degradé mais pas en danger.\nUn cercle de couleur verte signifie que la Platform est degradée et en danger.\nSurvolez les sgements du graphique pour plus de details.",
-  "dashboard_cpu_quantile_explanation_line1": "Affichage des statistiques sur l'utilisation du processeur de votre cluster pour les grands clusters.",
-  "dashboard_cpu_quantile_explanation_line2": "Q90 est le quantile 90, ce qui signifie que 90 % de vos nœuds utilisent moins de CPU que ce seuil.",
-  "dashboard_cpu_quantile_explanation_line3": "Q5 est le quantile 5, ce qui signifie que 5 % de vos nœuds utilisent moins de CPU que ce seuil.",
-  "dashboard_cpu_quantile_explanation_line4": "Si ces indicateurs divergent de la médiane, cela signifie que la charge CPU de votre cluster n'est pas correctement équilibrée.",
-  "dashboard_cpu_quantile_explanation_line5": "Le survol du graphique affiche les nœuds qui sont respectivement au-dessus et en dessous des seuils Q90 et Q5.",
+  "metric_quantile_explanation": "Affichage de statistiques des métriques du cluster lorsque plus de 8 nœuds sont déployés.\nQ90 est le percentile 90, ce qui signifie que 90% des nœuds consomment moins de ressources que ce seuil.\nQ5 est le percentile 5, ce qui signifie que 5% des nœuds consomment moins de ressources que ce seuil.\nSi ces indicateurs s'éloignent de la médiane, cela signifie que le cluster est mal équilibré.\nAfin de découvrir les nœuds qui sont respectivement au-dessus et en dessous des seuils Q90 et Q5, veuillez survoler le graphique.",
+  "network_quantile_explanation": "Affichage de statistiques de la bande passante utilisée du cluster lorsque plus de 8 nœuds sont déployés.\nQ90 est le percentile 90, ce qui signifie que 90% des nœuds consomment moins de bande passante que ce seuil.\nQ5 est le percentile 5, ce qui signifie que 5% des nœuds consomment moins de bande passante que ce seuil.\nSi ces indicateurs s'éloignent de la médiane, cela signifie que le cluster est mal équilibré.\nAfin de découvrir les nœuds qui sont respectivement au-dessus et en dessous des seuils Q90 et Q5, veuillez survoler le graphique.",
```

Comment:
I guess this is automated translation?
We won't be able to review this, translation is a very controlled field -- we do not know the french technical terms for Scality and/or the software industry. I cannot tell you if "90 est le percentile 90" is right, or not...
This must be reviewed by a translator // a reviewer (it's a real job :D ) 
I'll address this with Erwan and Thomas.
Techpubs will of course review the English in your PRs!

Reply:
> I guess this is automated translation?

That's harsh... Honestly, the sentence seem perfectly clear in French. The "percentile 90" may be reworded to "90ème percentile", though the former is still understandable.

Reply:
Actually, the English variant is worse, IMO: "badly equilibrated" sounds French :smirk:

Reply:
Automated translation isn't a bad word, I'm just saying that if you paid a translator for this, well... Change it :D 

As said, we will review the English with no problem. French is another job, it may be our mother language, but I personally do not want to review it as I don't feel legit enough to provide a qualitative translation / review (e.g. no knowledge on the technical terms, nor on the French technical writing norms)
My only expertise in this case is just to alert on that :)

Reply:
Hello @lucieleonard 
I understand your point and let's focus only on the English one for the moment.